python/kvikio/kvikio/_lib/libnvcomp_ll.pyx

```diff
@@ -1038,14 +1054,19 @@ class nvCompBatchAlgorithmDeflate(nvCompBatchAlgorithm):
 
     options: nvcompBatchedDeflateOpts_t
 
-    def __init__(self, algo: int = 0):
-        self.options = nvcompBatchedDeflateOpts_t(algo)
+    def __init__(self, algo: int = None):
+        if algo is None:
+            self.options = nvcompBatchedDeflateDefaultOpts
+        elif isinstance(algo, int):
+            self.options = nvcompBatchedDeflateOpts_t(algo)
+        else:
+            raise ValueError(f"Invalid value for algo: {algo}")
 
     def _get_comp_temp_size(
         self,
         size_t batch_size,
         size_t max_uncompressed_chunk_bytes,
-    ) -> (nvcompStatus_t, size_t):
+    ) -> tuple[nvcompStatus_t, size_t]:
```

Comment:
hmm, how did this ever work? I guess cython is less strict with parsing.

Reply:
Both are valid, the type format we had before is called a `ctuple` and has been around in Cython longer

Support for `typing.Tuple` is relatively new

Here's [an example in the docs]( https://cython.readthedocs.io/en/3.0.x/src/userguide/language_basics.html#types )